core/src/elements/ons-pull-hook.js

```diff
@@ -424,7 +436,8 @@ export default class PullHookElement extends BaseElement {
       this._gestureDetector = new GestureDetector(this._pageElement, {
         dragMinDistance: 1,
         dragDistanceCorrection: false,
-        dragLockToAxis: !this._dragLockDisabled
+        dragLockToAxis: !this._dragLockDisabled,
+        passive: !this._shouldFixScroll
```

Comment:
Can we set this to `passive: true` if we use the `overflow: hidden` scroll fix?

Reply:
@asialgearoid I don't think so, the pull-hook flickering issue is not related to the `overflow: hidden` thing on iOS